### PR TITLE
Create the Cody 1.16 release blog

### PR DESCRIPTION
**Pull request by iskyOS**
Adds the new blog file for the Cody v1.16.0 release post.

### CODE REVIEW

File: content/blogposts/2024/cody-vscode-1-16-0-release.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Changelog
+
+See the [changelog](https://github.com/sourcegraph/cody/releases/tag/vscode-v1.14.0) and [GitHub releases](https://github.com/sourcegraph/cody/releases) for a complete list of changes.
+
+
+## Thank you
+
+Cody wouldn’t be what it is without our amazing contributors 💖 A big thank you to everyone who contributed, filed issues, and sent us feedback.
+
+We value your feedback in our [support forum](https://community.sourcegraph.com/), [Discord](https://discord.com/servers/sourcegraph-969688426372825169) and [GitHub](https://github.com/sourcegraph/cody/discussions). Happy Codying!
```
Comment (jdorfman) [inline content/blogposts/2024/cody-vscode-1-16-0-release.md:104]: Remove GitHub Discussions. We no longer use it.

File: content/blogposts/2024/cody-vscode-1-16-0-release.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+We’ve exposed a new ContextMentionProvider API. You can use it to build new context sources that you can `@-mention` in chat. You can define `triggerPrefixes`, like `@npm:`, then use those prefixes to select context to pass to Cody.
+
+This API is off by default and only enabled when you set `cody.experimental.noodle` to `true` in VS Code settings. Note that this API and the context provider implementation are subject to change.
+
+[Read the pull request](https://github.com/sourcegraph/cody/pull/3883) for more details.
+
+
+## Changelog
+
+See the [changelog](https://github.com/sourcegraph/cody/releases/tag/vscode-v1.14.0) and [GitHub releases](https://github.com/sourcegraph/cody/releases) for a complete list of changes.
```
Comment (jdorfman) [inline content/blogposts/2024/cody-vscode-1-16-0-release.md:97]: `https://github.com/sourcegraph/cody/releases/tag/vscode-v1.16.0`

**Review by kalanchan — DISMISSED** [review]
looks great ty!!

**Review by jdorfman — CHANGES_REQUESTED** [review]
looks good, two link issues

File: content/blogposts/2024/cody-vscode-1-16-0-release.md (hunk 1 above)
Comment (iskyOS) [reply content/blogposts/2024/cody-vscode-1-16-0-release.md]: Fixed!

File: content/blogposts/2024/cody-vscode-1-16-0-release.md (hunk 2 above)
Comment (iskyOS) [reply content/blogposts/2024/cody-vscode-1-16-0-release.md]: Updated

**Review by jdorfman — APPROVED** [review]

**Pull request merged**